.babelrc

```diff
@@ -1,4 +1,18 @@
 {
-  "presets": ["react"],
-  "plugins": ["transform-es2015-modules-amd"]
+  "presets": [
+    "react"
+  ],
+  "plugins": [
+    "transform-es2015-modules-amd",
+    "check-es2015-constants",
```

Comment:
__issue:__ looks like you forgot to include these plugins with the proper version to ```package.json```